etc/db_schema_filemetadata_table.sql

```diff
@@ -8,6 +8,8 @@ CREATE TABLE FileMetadata(
 	uid INT NOT NULL,
 	cid INT NOT NULL,
 	creation_date DATETIME,
+    title VARCHAR(64) NOT NULL,
+    description VARCHAR(255) NOT NULL,
```

Comment:
These lines have spaces and the other lines have tabs. Mind making all lines have spaces?